client_test.go

```diff
@@ -19,6 +19,7 @@ import (
 	"time"
 
 	"github.com/dgrijalva/jwt-go"
+	"github.com/stretchr/testify/assert"
```

Comment:
I'm ok with this import, but then we need to rewrite all tests to use testify instead of `FailIfErr` and etc functions.
> FailIfError -> assert.NoError
> FailIfNotErr -> assert.Error
> FailIf -> assert.Fail
> AssertEquals -> assert.Equals
> AssertNotEquals -> assert.NotEqual

I didn't know about testify library at a time I implemented the unit tests.
But this can be and should be done in a separate pull request.

Reply:
Hm I know, I wanted to use your assert methods, but unfortunately the deep assert that is happening in there tells me my 1 is not a 1 and my 0 is not a 0. I understand that replacing everything is unrelated and a fairly huge task, but if you want to, I can also do that. What do you suggest?

Reply:
If you have a time, you can do it :) Or we can let it be as it is and replace it later. Or you can use `assert.` function in the new unit tests.

Reply:
I thought it would not be that complicated to replace it, but unfortunately it is because we have to distinguish between critical functions (such as create and delete) that build tests or tidy up and others that just have to be checked. If critical changes fail, I think we have to use `assert.FailNow()` or similar... I have started implementing but I would for now prefer to leave as is, if that is okay for you. Updated the README anyways!

Reply:
Thank you!